### ISSUE TITLE
[suggestion] hyperthermia/hypothermia rework

### ISSUE THREAD

**Issue by Biviho** [Charles445/SimpleDifficulty#12]
hi, currently the way hyperthermia and hypothermia work is that, when you fill a bar (or the bar is empty), you get an hunger like damage.
this is incredibly easy to just ignore with a bit of regen, making all the mechanics and interesting tools just negligable.
my suggestion is to rework them to be more of a spectrum of effect: for example, when the flame appear, you get hyperthermia 1 that give you thirst, when the flame start shaking you get hyperthermia 2 that also add slowness for example.

a cool thing could be to make this effect actually customizable in the config with each level giving you custom potion effects
something like:

[
hyperthermia,1,potioncore:vulnerability,1,
hyperthermia,1,minecraft:weakness,1,
hyperthermia,2,potioncore:vunerability,2, 
etc...
]

this would be something like: first level of hyperthermia apply vulnerability 1 and weakness 1 for the duration of hyperthermia 1, instead hyperthermia 2 apply only vulnerability 2 for the duration.

this is just an example, but something like that

**Comment by MrRedstoneA1** [comment]
True, regen works wonders in completely negating both hypothermia and hyperthermia. Maybe after the effect has been on for too long, it eventually drains all of your health very quickly, similar to having the wither effect?

**Comment by Charles445** [comment]
Not sure if this config option is in the latest version, but there's the option to make damage scale while it's not being taken care of
TemperatureDamageScaling and ThirstDamageScaling
They basically do just that, if you have the effects on for too long, they do more damage
I found even small amounts of temperature damage scaling to be lethal, since the mere act of recovering from cold temperatures ticks you very frequently.
If it's in the latest version, try it out, it might do the trick
